packages/roosterjs-editor-dom/lib/table/pasteTable.ts

```diff
@@ -0,0 +1,57 @@
+import findClosestElementAncestor from '../utils/findClosestElementAncestor';
+import moveChildNodes from '../utils/moveChildNodes';
+import VTable from './VTable';
+import { NodePosition, TableOperation } from 'roosterjs-editor-types';
+
+/**
+ * Pastes a table inside another, modifying the original to create a merged one
+ * @param root The table to paste into
+ * @param rootNodeToInsert A Node containing the table to be inserted
+ * @param position The position to paste the table
+ */
+export default function pasteTable(
+    root: HTMLElement,
+    rootNodeToInsert: HTMLTableElement,
+    position: NodePosition,
+    range: Range
+) {
+    let currentTd = findClosestElementAncestor(position.node, root, 'TD,TH');
+
+    // This is the table on the clipboard
+    let newTable = new VTable(rootNodeToInsert);
+    // This table is already on the editor
+    let currentTable = new VTable(<HTMLTableCellElement>currentTd);
```

Comment:
do we need to do null check here?

I think there is already code outside this function to get the TD node, you can just pass it in so that no need to do null check and type cast here

Reply:
If it returns null, then you should not create VTable, right?

But actually it is already check in adjustInsertPosition.ts, Line 132, so you just need to pass the result from there into this function, so no need to call findClosestElementAncestor again and no need to do null check.

Reply:
@Andres-CT98 any update to this comment?

Reply:
I removed the repeated call to findClosestElementAncestor and just pass the firsts result.